src/components/overlays/TheCreateNewTacticOverlay.vue

```diff
@@ -0,0 +1,149 @@
+          <v-col>
+            <v-card-subtitle>
+              <v-img v-if="tactic.map.icon" :src="tactic.map.icon" max-width="200px"/>
+            </v-card-subtitle>
+            <v-card-subtitle>{{ tactic.map.name }}</v-card-subtitle>
+          </v-col>
+        </v-row>
+      </v-card>
+    </v-dialog>
+</template>
+<script lang="ts">
+import Component, { mixins } from 'vue-class-component'
+import { EventBus } from '@/event-bus'
+import { RoomGetters } from '@/store/modules/room'
+import { Action, Getter } from 'vuex-class'
+import { Api } from '@/store/modules/types'
+import { StageActions } from '@/store/modules/stage'
+import { CustomStageConfig } from '@/util/PointerEventMapper'
+import { CanvasAction } from '@/store/modules/canvas'
+import { CanvasElement, CanvasElementHistory } from '@/types/Canvas'
+import TacticWatcher from '@/mixins/TacticWatcher'
+import uuid from 'uuid'
+import { AuthenticationGetters } from '@/store/modules/authentication'
+
+@Component({
+  name: 'TheCreateNewTacticOverlay',
+  data: () => ({
+    tactic: {
+      map: {
+        name: '',
+        icon: '',
+        desc: '',
+        width: 0,
+        height: 0,
+        ratio: 0,
+        id: 0
+      },
+      name: ''
+    },
+    search: ''
+  }),
+  mixins: [TacticWatcher]
+})
+export default class CreateNewTacticOverlay extends mixins(TacticWatcher) {
+  @Getter(`room/${RoomGetters.GAME_API}`) gameApi!: Api[]
+  @Action(`stage/${StageActions.SET_CONFIG}`) setConfig!: (config: CustomStageConfig) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT}`) setCanvasElements!: (canvasElements: CanvasElement[]) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT_HISTORY}`) setCanvasElementsHistory!: (canvasElements: CanvasElementHistory[]) => void
+
+  overlay = false
+  created () {
+    EventBus.$on('openCreateNewTacticOverlay', () => {
+      this.overlay = !this.overlay
+    })
+  }
+
+  get maps () {
+    const mapApi: Api[] = this.gameApi.filter((api: Api) => api.name === 'wows.encyclopedia.maps')
+    if (mapApi.length === 1) {
+      return mapApi[0].data
+    } else {
+      return false
+    }
+  }
+
+  $refs!: {
+    img: HTMLImageElement;
+    imgName: HTMLHeadingElement;
+  }
+
+  isDisabled (): boolean {
+    return !(this.$data.tactic.name && this.$data.tactic.map.icon)
+  }
+
+  // Need to do more to this, but we dont have the collection stuff created yet so this is temporary.
+  createTactic (): void {
+    if (!this.isDisabled()) {
+      this.newTactic({
+        id: uuid(),
+        name: this.$data.tactic.name,
+        collectionId: uuid(),
+        canvasElements: [],
+        canvasElementsHistory: [],
+        lockedBy: undefined,
+        createdBy: this.$store.getters[`authentication/${AuthenticationGetters.JWT}`].jti,
+        map: this.$data.tactic.map
+      })
+      this.overlay = false
+    }
+  }
+}
+</script>
+<style lang="scss" scoped>
+  .custom-overlay {
+    width: 700px;
```

Comment:
If possible, need better design, but fine by me for alpha/beta but not for 1.0.0 release.

Reply:
Dont worry, mobile users should not be able to ceate new tactics, i forgot to hide it on mobile

Reply:
In case we dont want it to show on mobile, i made some changes:
![image](https://user-images.githubusercontent.com/38263092/81477917-81a37680-921a-11ea-8249-388b85adcc68.png)